pkgs/by-name/fr/frida-tools/package.nix

```diff
@@ -6,12 +6,13 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "frida-tools";
-  version = "14.4.5";
+  version = "14.5.0";
   format = "pyproject";
 
   src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-sId91KB2qLasJHsfrS6Nfqctn0kCPS6ieNwtfheai8M=";
+    inherit version;
+    pname = lib.replaceString "-" "_" pname;
```

Comment:
Can confirm fetching with the dash does not work for this version but that they refer to the same package on pypi.

Nit: At this point just change it to `pname = "frida_tools";`, there is no typing to be saved and also no reason to so strongly tie the fetch pname to the actual package name.